inputs/demand/industry/capacity_of_industry_other_paper_flexibility_p2h_electricity.ad

```diff
@@ -15,7 +15,7 @@
       )
     )
 - priority = 0
-- max_value_gql = present:MAX(1.0,PRODUCT(DIVIDE(V(industry_useful_demand_for_other_paper_useable_heat,demand),(8760 * MJ_PER_MWH)),2.0))
+- max_value_gql = present:MAX(25.0,PRODUCT(DIVIDE(V(industry_useful_demand_for_other_paper_useable_heat,demand),(8760 * MJ_PER_MWH)),2.0))
```

Comment:
Why not use the same max value as for industry_chemical_refineries? If possible, I would use uniform max_values for uniform sliders.

Reply:
I have updated all industry flex p2h inputs to be at least 10000 MW, in line with the proposed change for industry_chemical_refineries.